### ISSUE TITLE
Nav-collapse question

### ISSUE THREAD

**Issue by SachaG** [erobit/meteor-accounts-ui-bootstrap-dropdown#7]
I've been having a problem where the log-in dropdown disappears at smaller window widths, as you can see here:

http://meteor-book-chapter14-2.meteor.com/

Is this the intended behavior of the `.nav-collapse` class? If it is, maybe it should be optional somehow?

**Comment by SachaG** [comment]
OK, so it seems I was missing the toggle button… that being said, even when adding it the button only toggles the dropdown link, not the actual dropdown itself. Am I doing something wrong, or is it just the way the dropdown is implemented?

**Comment by erobit** [comment]
Hi Sacha,

Issue is mrt doesn't have the latest. I'm having problems pushing to atmosphere.

/usr/local/lib/node_modules/meteorite/lib/atmosphere.js:44
        throw "Tag " + versionName + " doesn't exist in repository, please add
                                   ^
Tag vundefined doesn't exist in repository, please add!

**Comment by erobit** [comment]
Actually, there is a css media query / class causing you grief on smaller resolutions.

@media (max-width: 979px)
.nav-collapse, .nav-collapse.collapse {
height: 0;
overflow: hidden;
}

**Comment by SachaG** [comment]
Maybe @tmeasday can help you with the mrt issue? 

And for the media query I know, but I didn't change any of the default bootstrap code… so this is why it seems strange to me that the default behavior would be for the dropdown to disappear completely.

**Comment by erobit** [comment]
Hi Sacha,

I think I found the issue. Check my latest commit. I tested on the bootstrap site and noticed that the "nav-collapse collapse" css classes could be swapped out for "dropdown" instead - which keeps the items visible.

I'm guessing people can customize their navs how they choose - to display on smaller resolutions, or collapse. I think the default should be display anyhow. Ideally this should be configurable with a setting passed to the loginButtons template helper.

Hope this solves your issues for now, if I get enough feedback I'll consider modifying the package to make it more customizable.
